client/rpcserver/handlers.go

```diff
@@ -1080,7 +1080,8 @@ var helpMsgs = map[string]helpMsg{
 	toggleWalletStatusRoute: {
 		pwArgsShort: "appPass",
 		argsShort:   `assetID disable`,
-		cmdSummary:  `Disable or enable an existing wallet.`,
+		cmdSummary: `Disable or enable an existing wallet. When disabling a parent wallet, all
+		token wallets that depend on the wallet will be disabled.`,
```

Comment:
"parent wallet" is what we use in code, but may not me customary. How about something like "When disabling a chain's primary asset wallet, all token wallets for that chain will be disabled too."